cmd/simulacra/simulacra.go

```diff
@@ -204,9 +217,61 @@ func getConfigFromYaml(configPath string) (*Config, error) {
 	return &config, nil
 }
 
+// Parse the metadata image name with format 'projects/debian-cloud/global/images/debian-11-bullseye-v20230711'
+// and return the scope and image name.
+func parseMetadataImage(name string) (string, string, string, error) {
+	components := strings.Split(name, "/")
+	if len(components) < 5 {
+		return "", "", "", fmt.Errorf("image name from metadata must be of format 'projects/debian-cloud/global/images/debian-11-bullseye-v20230711' ")
+	}
+	imgProject := components[1]
+	scope := components[2]
+	image := components[4]
+	return imgProject, scope, image, nil
+}
+
+func getConfigFromDiagnosticOutput(outputDir string) (*Config, error) {
+	type Metadata struct {
+		Image string `json:"image"`
+	}
+
+	metadataFile, err := os.ReadFile(path.Join(outputDir, "vm_config.json"))
```

Comment:
Use `filepath.Join` instead for cross-platform compatibility. See [explanation in package docs](https://pkg.go.dev/path).